product_dimension/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Camptocamp SA
```

Comment:
You can remove copyright from `init` files https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py